main.go

```diff
@@ -31,9 +31,13 @@ func main() {
 	// Create a new logger which is used by all packages.
 	var newLogger micrologger.Logger
 	{
-		loggerConfig := micrologger.DefaultConfig()
-		loggerConfig.IOWriter = os.Stdout
-		newLogger, err = micrologger.New(loggerConfig)
+		c := micrologger.Config{
+			Caller:             micrologger.DefaultCaller,
+			IOWriter:           micrologger.DefaultIOWriter,
+			TimestampFormatter: micrologger.DefaultTimestampFormatter,
+		}
```

Comment:
It isn't needed, those values are set by default.

Reply:
https://github.com/giantswarm/micrologger/blob/2aaea11da0707ebb7f9fbabb07028a37b00efc16/logger.go#L24-L32

Reply:
I see. Cool. The zero values are `nil`. I was still in the operatorkit mode where we had to assign the default values for the rate wait manually.